code/game/machinery/vending.dm

```diff
@@ -149,7 +149,7 @@
 	del(wires) // qdel
 	wires = null
 	if(coin)
-		del(coin) // qdel
+		qdel(coin) // qdel

```

Comment:
The wires datum up there should be qdel'd, too, for if they're ever made queueable.